application/operator/NGSI-LD-operator/waterpumpstatus/config.json

```diff
@@ -0,0 +1,16 @@
+[{
```

Comment:
(Major)remove this use case its no more needed as waterpump is the running use case.